### PR TITLE
OADP-3525: Remove the name field from snapshotLocations spec

### PR DESCRIPTION
**Pull request by anarnold97**
### JIRA

* [OADP-3525](https://issues.redhat.com/browse/OADP-3525)

### VERSIONS

* OCP 4.12 → branch/enterprise-4.12
* OCP 4.13 → branch/enterprise-4.13
* OCP 4.14 → branch/enterprise-4.14
* OCP 4.15 → branch/enterprise-4.15
* OCP 4.16 → branch/enterprise-4.16

### Link to docs preview:

* [Installing the Data Protection Application 1.2 and earlier](https://72144--ocpdocs-pr.netlify.app/openshift-enterprise/latest/backup_and_restore/application_backup_and_restore/installing/installing-oadp-aws#oadp-installing-dpa-1-2-and-earlier_installing-oadp-aws) - module included elsewhere but preview will be valid for all assemblies

* [Creating profiles for different credentials](https://72144--ocpdocs-pr.netlify.app/openshift-enterprise/latest/backup_and_restore/application_backup_and_restore/installing/installing-oadp-aws#oadp-creating-default-secret_installing-oadp-aws) - module included elsewhere but preview will be valid for all assemblies

### QE review:
- [X] [QE has approved this change](https://github.com/openshift/openshift-docs/pull/72144#issuecomment-1982619664).

Additional information:

### CODE REVIEW

File: modules/oadp-installing-dpa-1-2-and-earlier.adoc [hunk 1]
```diff
@@ -69,8 +69,7 @@ spec:
       podConfig:
         nodeSelector: <node_selector> <4>
   backupLocations:
-    - name: default
-      velero:
+    - velero:
```
Comment (amastbau) [inline modules/oadp-installing-dpa-1-2-and-earlier.adoc:72]: name is not supported for  snapshotLocations but it if for backupLocations
please undo this change

**Review by amastbau — CHANGES_REQUESTED** [review]

**Comment by amastbau** [comment]
LGTM

File: modules/oadp-installing-dpa-1-2-and-earlier.adoc [hunk 2]
```diff
@@ -83,8 +83,7 @@ spec:
           key: cloud
           name: {credentials} <7>
   snapshotLocations: <8>
```
Comment (dfitzmau) [inline modules/oadp-installing-dpa-1-2-and-earlier.adoc:85]: Hi @anarnold97 . I see that [Installing the Data Protection Application 1.3](https://72144--ocpdocs-pr.netlify.app/openshift-enterprise/latest/backup_and_restore/application_backup_and_restore/installing/installing-oadp-aws#oadp-installing-dpa-1-3_installing-oadp-aws) section has a name entry in the YAML. OK?

**Comment by dfitzmau** [comment]
Hi @anarnold97 . Nice work. I added comments inline.

/remove-label peer-review-in-progress

/label peer-review-done

File: modules/oadp-installing-dpa-1-2-and-earlier.adoc (hunk 2 above)
Comment (anarnold97) [reply modules/oadp-installing-dpa-1-2-and-earlier.adoc:85]: Yes

File: modules/oadp-secrets-for-different-credentials.adoc [hunk 3]
```diff
@@ -63,8 +63,7 @@ spec:
           key: cloud
           name: {credentials}
   snapshotLocations:
-    - name: default
```
[the remark this answers is not in the document]
Comment (anarnold97) [reply modules/oadp-secrets-for-different-credentials.adoc:66]: @amastbau - please can I ask you to confirm. Thanks

[the remark this answers is not in the document]
Comment (anarnold97) [reply modules/oadp-secrets-for-different-credentials.adoc:66]: Yes it is needed. Thanks
